source/MRViewer/MRUIStyle.h

```diff
@@ -180,12 +180,11 @@ MRVIEWER_API bool beginCombo( const char* label, const std::string& text = "Not
 MRVIEWER_API void endCombo( bool showPreview = true );
 
 
-MRVIEWER_API bool sliderFloat( const char* label, float* v, float v_min, float v_max, const char* format = "%.3f", ImGuiSliderFlags flags = 0 );
-MRVIEWER_API bool sliderInt( const char* label, int* v, int v_min, int v_max, const char* format = "%d", ImGuiSliderFlags flags = 0 );
```

Comment:
where are they now?

Reply:
They shouldn't be used anymore, because we have the new templated `UI::slider()`.